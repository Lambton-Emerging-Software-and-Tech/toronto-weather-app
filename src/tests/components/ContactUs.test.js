import React from 'react';
import { shallow } from 'enzyme';
import ContactUs from '../../components/ContactUs';

test('should test ContactUs component', () => {
 const wrapper = shallow(<ContactUs />);
 expect(wrapper).toMatchSnapshot();
});