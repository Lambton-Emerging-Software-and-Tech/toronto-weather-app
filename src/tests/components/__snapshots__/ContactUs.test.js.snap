// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test ContactUs component 1`] = `
<div>
  This is ContactUs
</div>
`;
